### PR TITLE
WT-8523 Create a script that can output MongoDB tables into BSON format

### PR DESCRIPTION
**Pull request by mm-ng**

### CODE REVIEW

File: tools/mdb_to_wt.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import bson, codecs, pprint, subprocess, sys
+
+# Decodes a MongoDB file into a readable format.
+def util_usage():
+    print("Usage: mdb_to_wt <path_to_wt> filename")
```
Comment (kommiharibabu) [inline tools/mdb_to_wt.py:33]: The name of the script may not be correct? I got it as it converts something from MDB to WT. How about `wt_to_mdb_bson`?

Comment (mm-ng) [reply tools/mdb_to_wt.py:33]: @kommiharibabu My understanding is that none of our WT tests generate anything in BSON format, hence this script is only used for MDB files e.g. indexes and collections. Is that correct?

Comment (kommiharibabu) [reply tools/mdb_to_wt.py:33]: Yes, that's correct, none of the WT tests doesn't generate BSON format data and also I agree that this script is used to decode the data stored by MongoDB in WT. 

As this script is converting the data stored in WT to MDB understanding BSON format, I suggested that name.

Comment (mm-ng) [reply tools/mdb_to_wt.py:33]: Ah yes, I see what you mean. I will change the name to the name you suggested.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Review by jiechenbo — APPROVED** [review]
LGTM! Nice work

**Comment by yangbodong22011** [comment]
When I executed the script, I encountered `Permission denied`, but my directory and wt file were given `x` permissions.

If I shouldn’t report a problem here, please remind me.

```
[bodong.ybd@fanche-dev-11.238.151.74 /home/bodong.ybd/workdir/mongo]
$ls -l
total 36
-rwxr-xr-x 1 bodong.ybd users 32768 Jan  5 15:00 collection-0--3010260785252159448.wt
-rwxr-xr-x 1 bodong.ybd users  2989 Jan  5 15:26 wt_to_mdb_bson.py

[bodong.ybd@fanche-dev-11.238.151.74 /home/bodong.ybd/workdir/mongo]
$ls -ld .
drwxr-xr-x 2 bodong.ybd users 4096 Jan  5 15:28 .

[bodong.ybd@fanche-dev-11.238.151.74 /home/bodong.ybd/workdir/mongo]
$python wt_to_mdb_bson.py /home/bodong.ybd/workdir/mongo collection-0--3010260785252159448.wt
/home/bodong.ybd/workdir/mongo
collection-0--3010260785252159448.wt
Traceback (most recent call last):
  File "wt_to_mdb_bson.py", line 78, in <module>
    dump_mdb_file(wtpath, filename)
  File "wt_to_mdb_bson.py", line 60, in dump_mdb_file
    mdb_hex = subprocess.check_output([wtpath, "dump", "-x", "file:" + filename], universal_newlines=True)
  File "/usr/lib64/python2.7/subprocess.py", line 568, in check_output
    process = Popen(stdout=PIPE, *popenargs, **kwargs)
  File "/usr/lib64/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/lib64/python2.7/subprocess.py", line 1327, in _execute_child
    raise child_exception
OSError: [Errno 13] Permission denied

[bodong.ybd@fanche-dev-11.238.151.74 /home/bodong.ybd/workdir/mongo]
```

**Pull request merged**